rust/private/clippy.bzl

```diff
@@ -80,15 +80,15 @@ def _clippy_aspect_impl(target, ctx):
     clippy_marker = ctx.actions.declare_file(ctx.label.name + ".clippy.ok")
 
     args, env = construct_arguments(
-        ctx,
-        ctx.rule.attr,
-        ctx.file,
-        toolchain,
-        toolchain.clippy_driver.path,
-        cc_toolchain,
-        feature_configuration,
-        crate_info,
-        dep_info,
+        ctx = ctx,
+        attr = ctx.rule.attr,
```

Comment:
Let's name the attribute the same here and in get_linker_and_args

Reply:
Done